targets/windows/handle_container.go

```diff
@@ -22,10 +22,21 @@ import (
 )
 
 const (
-	defaultBaseImage = "mcr.microsoft.com/windows/nanoserver:1809"
+	winltsc2019 = "mcr.microsoft.com/windows/nanoserver:ltsc2019"
```

Comment:
Note: ltsc2019 and 1809 are the same images.

Reply:
maybe add a comment about this?